_posts/2021-01-11-GPS-serverless.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: 7 Steps to Create a Serverless Tracking Solution
+date: 2021-01-11
+tags: cloudformation serverless appsync dynamodb
+author: Shermayne Lee
+image:
+---
+
+While being a dog lover, I love spending most of my free time outdoors with my dog. However, I always have difficulty calling him back when he runs outside, and I hate when he is out of my sight. Therefore to solve a few challenges I faced when I am with him, I came out with a serverless GPS monitoring solution to track his movement.
```

Comment:
> Therefore to solve a few challenges I faced when I am with him, I came out with a serverless GPS monitoring solution to track his movement.

Are these challenges you haven't mentioned?
If so mention them. Perhaps also give examples. We love examples!